sphinx_git/__init__.py

```diff
@@ -175,7 +176,12 @@ def _build_markup(self, commits):
                 detailed_message = None
 
             item = nodes.list_item()
-            item += nodes.strong(text=message)
+            # choose detailed message style by detailed-message-strong option
+            if self.options.get('detailed-message-strong', False):
```

Comment:
I'd prefer for this to default to `True`, so that the current behaviour is retained by default.

Reply:
Done in last commit